gpkit/tests/t_constraints.py

```diff
@@ -125,8 +126,31 @@ def test_init(self):
         self.assertTrue(isinstance(sc, SignomialInequality))
         self.assertFalse(isinstance(sc, Posynomial))
 
-
-TESTS = [TestConstraint, TestMonomialEquality, TestSignomialInequality]
+class TestBreakdown(unittest.TestCase):
```

Comment:
Unfortunately putting the test here leads to it being tested once for every installed solver...